app/controllers/users_controller.rb

```diff
@@ -2,5 +2,6 @@
 class UsersController < ApplicationController
   def index
     @pagy, @users = pagy(User.where(params[:company_id]),  items: 5)
+    add_breadcrumb t('shared.users'), :company_users_path  
```

Comment:
Make it the first line of the method.
Add respond_to block
Add load_and_authorize

Reply:
done